src/Backend/OnboardingInsuranceAPI/Services/DataContext.cs

```diff
@@ -41,6 +49,36 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 ProfileImage = "https://rostupload.blob.core.windows.net/images/default.jpg"
             });
 
+            modelBuilder.Entity<CarInsuranceInfo>()
+            .ToContainer("Insurances")
+            .HasPartitionKey(o => o.Pid)
+            .HasData(
+            new List<CarInsuranceInfo>()
+            {
+                new CarInsuranceInfo
+                {
+                    Pid = "773ba94b-e9a8-4a1c-9d7e-655eb2f426d9",
+                    InsuranceId = "ca4f0600-9493-4d9a-8580-ee5a16119520",
+                    CarBarnd = "Škoda",
+                    CarType = "SportCar",
+                    Crashed = true,
+                    FirstOwner = true,
+                    InformationNote = "Confident driver",
+                    ExpirationDate = new DateOnly(2023, 03, 20),
+                    LastService = new DateOnly(2022, 01, 01),
+                    YearOfProduction = 2020,
+                    YearlyContribution = 4760
+                }
+            });
+
+            modelBuilder.Entity<CarInsuranceInfo>()
+            .Property(e => e.ExpirationDate)
+            .HasConversion(v => v.Value.ToString("yyyy-MM-dd"), v => DateOnly.Parse(v));
+
+            modelBuilder.Entity<CarInsuranceInfo>()
+            .Property(e => e.LastService)
+            .HasConversion(v => v.Value.ToString("yyyy-MM-dd"), v => DateOnly.Parse(v));
```

Comment:
You can do type conversions like this: 
![image](https://user-images.githubusercontent.com/89402322/186198656-c0c3eb63-e4ee-41da-9fc3-479c6b953171.png)
protected override void ConfigureConventions(ModelConfigurationBuilder configurationBuilder)
{
    configurationBuilder
        .Properties<Currency>()
        .HaveConversion<CurrencyConverter>();
}